cafebabel/articles/models.py

```diff
@@ -104,7 +103,12 @@ def save_from_request(self, request):
             self.delete_image()
         image = files.get('image')
         if image:
-            self.attach_image(image)
+            if image.filename == '':
+                raise ValidationError('No selected file.')
+            if allowed_file(image.filename):
```

Comment:
I'd rather read `if not allowed_file… raise ValidationError` so that tests raise error and the nominal case is straighforward